.gitignore

```diff
@@ -1,4 +1,5 @@
-.idea/*
-*.DS_STORE
-*/.DS_STORE
-*/*/.DS_STORE
```

Comment:
.idea/* is needed to ignore PHPStorm files, and the .DS_STORE files appear at multiple levels - the single line of .DS_STORE won't ignore them all. I believe these lines should still be there.

Reply:
joseph clarified on the DS_Store thing, so nm on that.

Reply:
To clarify -

To exclude a directory with a given name, anywhere in the repo, list the name, followed by `/`:

```
.DS_STORE/
```

If you use a certain IDE, it's a good idea to put the IDE files (`.idea/`, `.vscode/`) into a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).